share/scripts/inverse/csg_table

```diff
@@ -96,6 +96,10 @@ postadd overwrite postadd_overwrite.sh
 postadd plot postadd_plot.sh
 postadd average postadd_average.sh
 
+# smoothing for updated potential to allow forgetting noise from previous iterations (rdfs)
+# the script is the same as postupd_smooth.sh
+postadd smooth postadd_smooth.sh
```

Comment:
Sorry for the late reply. I agree we should discuss this in a separate issue. 

Regarding the question of CG potential smoothing, I agree that due to CG updates may contain noise due to insufficient sampling. But, as per my understanding, I believe, it is more appropriate to address the noise in the update, i.e. dU, either by smoothing or (more accurately) by better sampling. As @junghans mentioned, smoothing the updated CG potential may inaccurately remove important features in the CG potential.